cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -606,6 +606,39 @@ func (ht *hcsTask) DumpGuestStacks(ctx context.Context) string {
 	return ""
 }
 
+// should only be called if hcs task is WCOW
```

Comment:
We dont run lint like we should on this repo but this would fail. A comment needs to say "// <method-name> ..."

Reply:
Why do we need the method actually? I figured it was for code reuse but there is only one caller.

Reply:
This was just to make the code clearer in the Stats function.